setup.py

```diff
@@ -1,8 +1,6 @@
 #! /usr/bin/env python
 
-# Copyright (C) 2008 Cournapeau David <cournape@gmail.com>
-# Copyright (C) 2009 Nathaniel Smith <njs@pobox.com>
-# Copyright (C) 2016 Antony Lee <anntzer.lee@gmail.com>
+# Copyright (C) 2008-2016 The scikit-sparse developers
```

Comment:
Don't you think that it is somehow double to mention some of the developers here and (all) again in ``overview.rst``?